entc/gen/template/meta.tmpl

```diff
@@ -100,7 +100,6 @@ var (
 		{{ $enum := trimPackage $f.Type.String $.Package }}
 		// {{ $enum }} defines the type for the {{ $f.Name }} enum field.
 		type {{ $enum }} string
-
```

Comment:
Can you add back please this line? Thanks

Reply:
Added back :)